src/agent/ncp_openthread.hpp

```diff
@@ -117,9 +125,9 @@ class ControllerOpenThread : public Controller
      * @retval  OTBR_ERROR_ERRNO        Failed to request the event.
      *
      */
-    virtual otbrError RequestEvent(int aEvent);
+    otbrError RequestEvent(int aEvent) override;
 
-    virtual ~ControllerOpenThread(void);
+    ~ControllerOpenThread(void);
```

Comment:
This class has custom destructor but no copy/move constructors, maybe a sign of issue.